features/step_definitions/common_steps.rb

```diff
@@ -506,8 +506,9 @@
     page.find_by_id(
       page.all(
         :xpath,
-        "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']").sample["aria-controls"]
-    ).text
+        "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']").sample["aria-controls"],
+        visible: :true
```

Comment:
`visible: true` is the default, I think we wanted this `visible: :all` is what we wanted for Chrome

```suggestion
        visible: :all
```